app/controllers/comment_votes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentVotesController < ApplicationController
+  def create
+    @comment_vote = CommentVote.where(comment_id: params[:comment_id], user_id: current_user.id).first_or_create
```

Comment:
This is neat, but you might want to give user feedback about whether they'd already voted?